pkg/notes/options.go

```diff
@@ -151,5 +159,32 @@ func (o *Options) ValidateAndFinish() error {
 		}
 	}
 
+	// TODO: It would be nice to validate the go template early (e.g. parse it).
+	// Override template path when string template is given.
+	if o.Template != "" {
+		o.TemplatePath = ""
+	}
+
+	if o.TemplatePath != "" {
+		b, err := ioutil.ReadFile(o.TemplatePath)
```

Comment:
I don't think reading the file should be the responsibility of the `options` struct? 🤷‍♂

But I guess we could also hash that out in https://github.com/kubernetes/release/issues/1016